src/wms/src/main/java/org/geoserver/wms/capabilities/GetCapabilitiesTransformer.java

```diff
@@ -1150,14 +1150,14 @@ protected void handleLayer(final LayerInfo layer, boolean isRoot) throws IOExcep
         }
 
         private void handleCommonStyleElements(StyleInfo defaultStyle) {
+            element("Name", defaultStyle.prefixedName());
```

Comment:
Moving the element name seems ... too easy :)

Is this same code used by WMS 1.1.1 and WMS 1.3.0? With our luck they have a different order or something terrible.

Reply:
aside: This is an excellent fix; especially if it let the project pass cite tests.